src/betterproto/__init__.py

```diff
@@ -756,12 +751,13 @@ def parse(self: T, data: bytes) -> T:
         self._serialized_on_wire = True
 
         for parsed in parse_fields(data):
-            field_name = self._betterproto.field_name_by_number.get(parsed.number)
+            proto_meta = self._betterproto
```

Comment:
Is it possible to move this outside the for loop?

Reply:
I was gonna remove it with the same rational as #140

Reply:
Actually maybe here it is ok

Reply:
Anywhere that attributes are being traversed inside loops is an opportunity for a small, easy performance optimization

If it was not inside a loop then I wouldn't bother